src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  width: $container-size;
+  padding: $container-padding;
```

Comment:
Ensure that the variables `$container-size` and `$container-padding` are defined in the 'variables' file or elsewhere in this SASS file. Without these definitions, the compilation will fail.